packages/datadog-plugin-apollo/src/gateway/postprocessing.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+const TracingPlugin = require('../../../dd-trace/src/plugins/tracing')
+const { storage } = require('../../../datadog-core')
+
+class ApolloGatewayPostProcessingPlugin extends TracingPlugin {
+  static get id () { return 'apollo.gateway' }
+  static get operation () { return 'postprocessing' }
+  static get type () { return 'web' }
+  static get kind () { return 'server' }
+  static get prefix () {
+    return 'tracing:apm:apollo:gateway:postprocessing'
+  }
+
+  bindStart (ctx) {
```

Comment:
This same code seems to be repeated in multiple plugins. Maybe it would be worth it adding some sort of base plugin specific to Apollo? Then this class could basically just become:

```js
class ApolloGatewayPostProcessingPlugin extends ApolloBasePlugin {
  static get operation () { return 'postprocessing' }
}
```

Reply:
A helper around service naming would make this possible for `gateway.fetch` as well.